bot/cogs/timeleft.py

```diff
@@ -0,0 +1,23 @@
+from discord.ext import commands
+from dateutil.relativedelta import relativedelta
+import logging
+import aiohttp
+import discord
+import datetime
+
+bot = commands.Bot(command_prefix='.', description='pr bot')

```

Comment:
Please remove the new bot instance instantiation. The bot is initialized elsewhere and this is unnecessary.